analysis/README.md

```diff
@@ -20,6 +20,7 @@ Further analysis can be returned by providing `:analysis` with a map of options:
 ```
 
 - `:locals`: when truthy return `:locals` and `:local-usages` described below
+- `:keywords`: when truthy return `:keywords` described below
```

Comment:
Can you also comment this out using HTML comments, also the docs below?
I think the docs are good, but let's not expose this feature to others until we are sure this is the right choice (everything in one list, no separation of defs, etc).

Reply:
@snoe ^